packages/openapi-typescript/src/load.ts

```diff
@@ -173,7 +173,7 @@ export default async function load(schema: URL | Subschema | Readable, options:
   else if (typeof schema === "object") {
     options.schemas[schemaID] = {
       hint: "OpenAPI3",
-      schema: schema as any,
+      schema: JSON.parse(JSON.stringify(schema)), // create deep clone of inline schema (don’t mutate)
```

Comment:
So funny enough, after some lazy performance testing in V8, `JSON.parse(JSON.stringify(…))` was faster than `structuredClone()`. Which I guess makes sense with JSON-serializable data that we’re working with.

This is probably not that much of a hack as I originally thought it was. And this only runs in the Node.js API (only when needed)